### PR TITLE
Added more schools

### PR DESCRIPTION
**Pull request by GuillermoFidalgo**
@graeme-a-stewart  
I have tried to follow the instructions  as best as I can, however if there is an error I would appreciate any help.

### CODE REVIEW

File: Schools/_posts/2018-09-25-AEPSHEP.md [hunk 1]
```diff
@@ -1,4 +1,12 @@
 ---
+<<<<<<< HEAD
```
Comment (graeme-a-stewart) [inline Schools/_posts/2018-09-25-AEPSHEP.md]: Something went really wrong here - this is a conflict marker...

**Comment by graeme-a-stewart** [comment]
@GuillermoFidalgo can you send me the results of running:

```
git remote -v show
git status
git branch -rv
```

Then I can provide some more concrete advice on how to simplify your state.

**Comment by GuillermoFidalgo** [comment]
image

File: Schools/_posts/2018-09-25-AEPSHEP.md (hunk 1 above)
Comment (GuillermoFidalgo) [reply Schools/_posts/2018-09-25-AEPSHEP.md]: I have fixed the conflict.

**Comment by graeme-a-stewart** [comment]
Hi @GuillermoFidalgo 

OK, here's a recipe that should get you out of the mess. As I said the basic problem is that you edited files on your own `master` branch, but that one you should always keep pristine and in sync with `upstream/master`.

```
git fetch --all # Make sure you're up to date with upstream
git checkout -b messy master # Create a new branch called "messy" that's a copy of your current master branch (safe place for all the changes made)
git checkout master # Switch to the messed up master branch
git reset --hard upstream/master # Force the local master branch to be the same as the clean upstream master
git push -f # Push the changes into GitHub (-f needed as the local copy diverges from the GH copy)
```

This has recovered your `master` branch to what it should be, and saved the current work in `messy`.

Now you need to create a new working branch for the actual changes, which you'll take from `messy`...

```
git checkout -b school-update upstream/master # Create a new branch and checkout, using upstream/master as the "parent"
git diff HEAD messy -- Schools > changes.patch # Get the per-file changes between messy and the new branch, but only for changed files in the Schools path (this simplifies things a lot)
git apply changes.patch # Now take those changes and apply them to this branch
git add <all the files that you actually want updated>
git commit
git push -u origin school-update
```

Now you can create a *new* pull request from the `school-update` branch. This one we should just close.

In the future always use the workflow `git checkout -b [NEW_BRANCH] upstream/master` so that you never need to merge or rebase the branch you are on. Follow the instructions in <https://hepsoftwarefoundation.org/github-beginners.html>.

BTW, I find <https://www.sourcetreeapp.com> a really brilliant git GUI - I use it all the time and only rarely need the command line.

**Pull request closed without merging**